README.md

```diff
@@ -132,6 +132,18 @@ pip install -e .[dev]
 ```
 </details>
 
+<details>
+<summary>(Optional) Enable PyTorch support</summary>
+
+You can enjoy the PyTorch XAI feature if the PyTorch is installed along with the OpenVINO XAI.
+
+```bash
+# Install PyTorch (CPU version as example)
+pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
```

Comment:
Do we need `torchvision` and `torchaudio`?

Reply:
Oops, forgot to remove after copying from the source. Removed in https://github.com/openvinotoolkit/openvino_xai/pull/68/commits/5a12594d9e49581e83f7afef4024616f434dc374.
Thanks!